service/controller/legacy/v29patch1/templates/cloudformation/tccp/record_sets.go

```diff
@@ -37,7 +37,7 @@ const RecordSets = `
       Name: 'api.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
       HostedZoneId: !Ref 'InternalHostedZone'
       Type: A
-  EtcdInternalRecordSet:
+  EtcdRecordSet:
```

Comment:
Can we keep this as `EtcdInternalRecordSet` Or will this also break?

Reply:
the problem is the existing `8.4.0` have the `EtcdRecordSet ` pointing to the internal, the order of CF is adding first the other recordset and fails because it exists. Not sure how to solve it without keeping the name

Reply:
Okay, I see. Let's keep the name for now and see if we can solve it otherwise later on.